src/domain/synthetics/trade/useTradeboxState.ts

```diff
@@ -156,9 +162,9 @@ export function useTradeboxState(
       setStoredOptionsOnChain({
         ...INITIAL_SYNTHETICS_TRADE_OPTIONS_STATE,
         markets: {
-          [market.marketTokenAddress]: {
-            long: market.longTokenAddress,
-            short: market.shortTokenAddress,
+          [market.indexTokenAddress]: {
+            long: market.marketTokenAddress,
+            short: market.marketTokenAddress,
```

Comment:
This seems strange
could you please explain why long and short otken are now the same market token address

Reply:
It all looks like we have a mistake in the prod codebase here. Since this is a part of the tradebox state, it does not store long and short tokens for the market. In fact, these are: "selected market for the Long tab" and "selected market for the Short tab."

If you look at how the markets property is used in other places, it becomes clear that the data structure there is:
```
{
  [indexTokenAdress]: {
    long: marketTokenAddressForLongs,
    short: marketTokenAddressForShorts,
  }
}
```
and only here it was different, which caused the bug.


Indeed, it's a confusing place. Thanks for pointing it out.

Reply:
ok